.github/workflows/helm-chart-ci.yaml

```diff
@@ -33,6 +33,28 @@ jobs:
       - name: Verify Docs updated
         run: ./helm-docs.sh
 
+  ensure-image-overrides:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.4.0
+        with:
+          fetch-depth: 0
```

Comment:
Only required for ct.

In this job you can remove the with block. As we don't need the other commits here.